### PR TITLE
Broadcast deploy updates

### PR DESCRIPTION
**Pull request by quartata**

### CODE REVIEW

**Comment by ArtOfCode-** [comment]
The `stream_from` calls in here are the legacy API; ideally, this should target the current API. See [`app/models/concerns/websocket.rb`](https://github.com/Charcoal-SE/metasmoke/blob/d969fe914a536c611df30bfeaa2ba7dcd1ab444a/app/models/concerns/websocket.rb) for how that's done - I'd suggest making channel `smokey_repos_update`, which implies `event_type: 'update', event_class: 'SmokeyRepo'`, and `object` is whatever data you want to send.

**Comment by quartata** [comment]
Oh, by the way: is there anything sensitive in params? I didn't think this particular webhook is signed or anything since it doesn't have :verify_github

**Comment by ArtOfCode-** [comment]
I dunno, what's in params?

**Comment by quartata** [comment]
@ArtOfCode- looking at the comment you left it doesn't look like it's anything account related, so it should be fine https://developer.github.com/v3/activity/events/types/#webhook-payload-example-26

**Comment by quartata** [comment]
I'll take a look at the new API (which I was not aware of at all, when did that change?) now.

**Comment by ArtOfCode-** [comment]
3 or 4 months ago :P

**Comment by quartata** [comment]
Oh, OK. I haven't done anything serious in Rails in a long time.

File: app/controllers/github_controller.rb [hunk 1]
```diff
@@ -173,6 +173,7 @@ def update_deploy_to_master
     # See https://developer.github.com/v3/activity/events/types/#webhook-payload-example-26
     # for what’s in `params`
     ActionCable.server.broadcast 'smokedetector_messages', deploy_updated: params
+    ApiChannel.broadcast_to 'api_deploy_update', event_type: 'update', event_class: 'Ref', object: params
```
Comment (ArtOfCode-) [inline app/controllers/github_controller.rb]: `channel` should match `event_class.pluralize.underscore` - not sure if it _actually_ makes a difference, but good to be consistent

**Pull request merged**